spec/i18n_spec.rb

```diff
@@ -219,7 +219,7 @@ def html_entities?(str)
 
   def likely_html_interpolation?(str)
     str.scan(I18n::INTERPOLATION_PATTERN).flatten.compact.any? do |key|
-      key =~ /html/
+      key.include? 'html'
```

Comment:
style nit, I'd use parens
```suggestion
      key.include?('html')
```